Firestore/core/src/remote/bloom_filter.h

```diff
@@ -87,23 +104,15 @@ class BloomFilter final {
   /** Return whether the bit at the given index in the bitmap is set to 1. */
   bool IsBitSet(int32_t index) const;
 
-  /**
-   * The number of bits in the bloom filter. Guaranteed to be non-negative, and
-   * less than the max number of bits `bitmap_` can represent, i.e.,
-   * bitmap_.size() * 8.
-   */
   int32_t bit_count_ = 0;
 
-  /**
-   * The number of hash functions used to construct the filter. Guaranteed to
-   * be non-negative.
-   */
   int32_t hash_count_ = 0;
 
-  /** Bloom filter's bitmap. */
   std::vector<uint8_t> bitmap_;
 };
 
+bool operator==(const BloomFilter& lhs, const BloomFilter& rhs);
```

Comment:
Add `operator!=()` as well, for symmetry. It's very simple, for example: https://github.com/firebase/firebase-ios-sdk/blob/b2383a9d3ca37b6a9e99c5fd9b75abdcf5a497c8/Firestore/core/src/model/transform_operation.h#L226-L229